src_py/sysfont.py

```diff
@@ -156,18 +160,19 @@ def _font_finder_darwin():
         "/Library/Fonts",
         "/Network/Library/Fonts",
         "/System/Library/Fonts",
-        "/System/Library/Fonts/Supplemental",
+        "/System/Library/Fonts/Supplemental"
```

Comment:
This change (and the others like it) is unnecessary and will probably fail our auto formatter `black`.